### DISCUSSION TITLE
键翻译后把中文标点符号替换成英文的，这样不影响 javadoc 的语法

### DISCUSSION THREAD

**Discussion by Lzw2016** [YiiGuxing/TranslationPlugin#3102] [Ideas]
如：
`Customization hook to create a {@link LettuceSubscription}.` 翻译后变成 `自定义挂钩以创建｛@link LettuceSubscription｝。` 
其中 "｛","｝","。" 是中午标点符号，是的 javadoc 失效，建议加一个配置能自动替换翻译后的字符

**Comment by YiiGuxing** [comment]
想要好的文档翻译效果，还是要谷歌翻译、微软翻译 和 阿里翻译。其他的可以看看 #1209 ，也可提升一点体验。

**Reply by Lzw2016** [reply]
我尝试过谷歌翻译、微软翻译 和 阿里翻译 对于我来说效果最好的是 **谷歌翻译** 可是谷歌翻译好像有额度限制，用一会就不能翻译成功了，要等几小时或者第二天才能用。

**Reply by YiiGuxing** [reply]
这种情况应该是谷歌翻译限制了你的IP，导致这个问题的可能原因是使用的频率过高了。

**Reply by Lzw2016** [reply]
我知道的呀，我可以付费的，但没有付费入口

**Reply by YiiGuxing** [reply]
谷歌翻译付费版本我是有计划接入的，但是现在任务堆积得太多了，而且个人的精力很有限，根本没有时间来做，我要优先处理免费方案的，排不上啊。。。

**Reply by Lzw2016** [reply]
那就先排个计划呗，谢谢啦
